manifests/config/global/securityroles.pp

```diff
@@ -0,0 +1,18 @@
+#
+# Author: Zoltan Lanyi <zoltan.lanyi@gmail.com>
+# Date  : 06.06.2016
+# 
+
+define rundeck::config::global::securityroles ($security_role = $title) {
+  augeas { "rundeck/web.xml/security-role/role-name/$security_role":
```

Comment:
there are a few highlevel modules that provide augeasproviders: https://forge.puppet.com/herculesteam

I *think* he means them. Maybe they provide a cleaner solution to handle the augeas stuff here?